src/types/viewport.cpp

```diff
@@ -137,6 +137,17 @@ COORD Viewport::Origin() const noexcept
     return { Left(), Top() };
 }
 
+// Method Description:
+// - For Accessibility, get a coord representing the end of this viewport in inclusive terms.
+// Arguments:
+// - <none>
+// Return Value:
+// - the coordinates of this viewport's end.
+COORD Viewport::EndInclusive() const noexcept

```

Comment:
Zis is not .. this.. the.. hmm. okay. I see. Okay. I do not love this name at al. I understand why I keep thinking you had it backwards.

_why is it called End Inclusive but it returns Bottom Exclusive_

Reply:
Renaming to `EndExclusive`. Not sure what I was on on that one.

Reply:
Copying comment from Griese below since it's fairly relevant:

This is needed to represent an exclusive endpoint in UiaTextRange that includes the last COORD's text in the buffer at (RightInclusive(), BottomInclusive())

So if we had a text range for the entire text buffer, it would be composed of Origin (0,0) and this (0, BottomExclusive()). Since the end is exclusive.